### PR TITLE
Replace all occurrences of Person with Patient

### LINKED ISSUE #39: Rename Person to Patient
A `Person` in Covigent may refer to either a staff member or a patient. To remove that ambiguity, let's replace all references to `Person` with 'Patient'.

### PR DESCRIPTION
**Pull request by w-yeehong**
Since we are handling `Patient`, we should replace all references to `Person` with `Patient`.

Would need to check whether I missed out anything.

### CODE REVIEW

File: src/test/data/JsonAddressBookStorageTest/invalidAndValidPatientAddressBook.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "persons": [ {
-    "name": "Valid Person",
+  "patients": [ {
+    "name": "Valid Patient",
     "phone": "9482424",
     "email": "hans@example.com",
```
Comment (LeeMingDe) [inline src/test/data/JsonAddressBookStorageTest/invalidAndValidPatientAddressBook.json]: hmm, i thought that this field not supposed to be here anymore?

**Review by LeeMingDe — APPROVED** [review]
LGTM

File: src/test/data/JsonAddressBookStorageTest/invalidAndValidPatientAddressBook.json (hunk 1 above)
Comment (w-yeehong) [reply src/test/data/JsonAddressBookStorageTest/invalidAndValidPatientAddressBook.json]: Agreed! Let's remove the additional fields.

**Review by raymondge — APPROVED** [review]
Nice Work! It's absolutely more covigent now!

**Comment by w-yeehong** [comment]
While not ideal, I decided to force push to my own repo in order to squash the commits that contained only minor changes for CheckStyle compliance.

I will be more careful when pushing to my repo in the future.

**Pull request merged**